server/routes/index.ts

```diff
@@ -5,14 +5,18 @@ import ConfirmArrivalController from './confirmArrivalController'
 import CheckAnswersController from './checkAnswersController'
 import ConfirmAddedToRollController from './confirmAddedToRollController'
 
+import authorisationForUrlMiddleware from '../middleware/authorisationForUrlMiddleware'
 import asyncMiddleware from '../middleware/asyncMiddleware'
 import { Services } from '../services'
 import TemporaryAbsencesController from './temporaryAbsencesController'
+import Role from '../authentication/role'
 
 export default function routes(services: Services): Router {
   const router = express.Router()
-  const get = (path: string, handler: RequestHandler) => router.get(path, asyncMiddleware(handler))
-  const post = (path: string, handler: RequestHandler) => router.post(path, asyncMiddleware(handler))
+  const get = (path: string, handler: RequestHandler, authorisedRoles?: Role[]) =>
+    router.get(path, authorisationForUrlMiddleware(authorisedRoles), asyncMiddleware(handler))
```

Comment:
I thought it made sense to have authorisation middleware specific for the URL. Makes it clear who can access what routes. Not seen this pattern in other services so happy for criticisms on this 👍